encoding/layout/infix.rs

```diff
@@ -27,7 +27,7 @@ pub enum Infix {
     PropertyLabel,
     PropertyValueType,
     PropertyOrdering,
-    PropertyOverride,
+    PropertyIsAbstract,
```

Comment:
Why is abstract a PropertyIsAbstract and also an PropertyAnnotationAbstract below?

Reply:
Beeeecaaaauuseee I forgot to remove it! It's not needed, thanks